cmd/aries-js-worker/src/aries.js

```diff
@@ -176,90 +176,255 @@ const Aries = function(opts) {
+
+            /**
+             * Sends a message to destination.
+             *
+             * @param req - json document
+             * @returns {Promise<String>}
+             */
+            sendNewMessage: async function (req) {
                 return invoke(aw, pending,  this.pkgname, "SendNewMessage", text, "timeout while sending new message")
             },
-            sendReplyMessage: async function (text) {
+
+            /**
+             * Sends a reply to an existing message.
+             *
+             * @param req - json document
+             * @returns {Promise<String>}
+             */
+            sendReplyMessage: async function (req) {
                 return invoke(aw, pending,  this.pkgname, "SendReplyMessage", text, "timeout while sending reply message")
             }
         },
 
+        /**
+         * VDRI methods - Refer to [OpenAPI spec](docs/rest/openapi_spec.md#generate-openapi-spec) for
+         * input params and output return json values.
+         */
         vdri: {
             pkgname: "vdri",
-            createPublicDID: async function (text) {
+
+            /**
+             * Creates a new Public DID.
+             *
+             * @param req - json document
+             * @returns {Promise<String>}
+             */
+            createPublicDID: async function (req) {
                 return invoke(aw, pending,  this.pkgname, "CreatePublicDID", text, "timeout while creating public DID")
             },
         },
 
+        /**
+         * Router methods - Refer to [OpenAPI spec](docs/rest/openapi_spec.md#generate-openapi-spec) for
+         * input params and output return json values.
+         */
         router: {
             pkgname: "router",
-            register: async function (text) {
+
+            /**
+             * Registers an agent with the router.
+             *
+             * @param req - json document
+             * @returns {Promise<String>}
+             */
+            register: async function (req) {
                 return invoke(aw, pending,  this.pkgname, "Register", text, "timeout while registering router")
             },
+
+            /**
+             * Unregisters an agent with the router.
+             *
+             * @returns {Promise<String>}
+             */
             unregister: async function () {
                 return invoke(aw, pending,  this.pkgname, "Unregister", "{}", "timeout while registering router")
             },
+
+            /**
+             * Retrieves the router connection id.
+             *
+             * @returns {Promise<String>}
+             */
             getConnection: async function () {
                 return invoke(aw, pending,  this.pkgname, "GetConnection", "{}", "timeout while fetching router connection id")
             }
         },
 
+        /**
+         * Verifiable methods related to credentials and presentations - Refer to [OpenAPI spec](docs/rest/openapi_spec.md#generate-openapi-spec) for
```

Comment:
TODO: should these functions be under vdri? (it's a bigger question than this PR).

Reply:
Created an issue https://github.com/hyperledger/aries-framework-go/issues/1367.